index.html

```diff
@@ -25,6 +25,12 @@
                 <div class="rectangle"></div>
             </div>
         </a>
```

Comment:
- [ ] OPTIONAL:  I have noticed that the page has a big overflow on the right side (white space) when I check with developer tools. I need to scroll the page right and left to see the contents. Please check the video below and address this issue: 

https://user-images.githubusercontent.com/77038610/161054355-97b2a34f-4fec-4afb-bb6e-5bf4249df280.mp4

Please use chrome developer tools and debug.

Reply:
Thanks for pointing out this issue. I will definitely give it a go over the weekend.  However, I dad implement only 2 breakpoints for the project, mobile (230px - breakpoint 480px) and desktop (1440px - breakpoint 780px). This was in line with project requirements.